src/main/java/duke/exec/Add.java

```diff
@@ -0,0 +1,39 @@
+package duke.exec;
+
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public class Add implements Executable {
+
+    // constants
+    private static final String TASK_COUNT_FRONT = "Now you have ";
+    private static final String TASK_COUNT_END = " task(s) in the list.";
+
+    private static final String ADDED_MESSAGE = "Got it. I've added this task:";
+
+    // instance variables
+    private Task toAdd;
+
+    // constructor
+    public Add(Task task) {
+        this.toAdd = task;
+    }
+
+    /**
+     * When run, adds a task to the given list of tasks
```

Comment:
Perhaps can start with a verb like "Adds a task" that explains what the method does?